### DISCUSSION TITLE
강의에서 소켓을 파일로 표현하는데 이게 강의에서 비유한건지 아니면 실제로 그렇게 추상화하는지 궁금합니다.

### DISCUSSION THREAD

**Discussion by kyY00n** [woowacourse-study/2023-tal-study#24] [Q&A]
제목이 내용입니다.

**Comment by yeonkkk** [comment]
[이 글](https://velog.io/@hidaehyunlee/%EC%BB%B4%ED%93%A8%ED%84%B0-%EA%B5%AC%EC%A1%B0%EB%A5%BC-%ED%86%B5%ED%95%B4-%EC%9D%B4%ED%95%B4%ED%95%98%EB%8A%94-%ED%8C%8C%EC%9D%BCFile%EA%B3%BC-%EC%86%8C%EC%BC%93Socket)을 참고해보면 좋을 것 같아요!

**Reply by yeonkkk** [reply]
https://www.inflearn.com/questions/866611/file%EA%B3%BC-socket%EC%97%90-%EB%8C%80%ED%95%B4%EC%84%9C

이 것도 같이 보면 좋을 것 같습니다!
